presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -369,4 +399,87 @@ public static LongWriteFunction timestampWriteFunction()
         }
         return Optional.empty();
     }
+
+    public static Object jdbcObjectToPrestoNative(Object jdbcObject, Type prestoType)
+    {
+        if (jdbcObject == null) {
+            return null;
+        }
+        else if (prestoType instanceof DecimalType) {
+            DecimalType decimalType = (DecimalType) prestoType;
+            BigDecimal value = (BigDecimal) jdbcObject;
+            if (decimalType.isShort()) {
+                return encodeShortScaledValue(value, decimalType.getScale());
+            }
+            else {
+                return encodeScaledValue(value, decimalType.getScale());
+            }
+        }
+        else if (REAL.equals(prestoType)) {
+            return floatToRawIntBits((float) jdbcObject);
+        }
+        else if (jdbcObject instanceof Date) {
+            return toLong((Date) jdbcObject);
+        }
+        else if (jdbcObject instanceof Time) {
+            return toLong((Time) jdbcObject);
+        }
+        else if (jdbcObject instanceof Timestamp) {
+            return ((Timestamp) jdbcObject).getTime();
+        }
+        else {
+            return jdbcObject;
+        }
+    }
+
+    public static Object prestoNativeToJdbcObject(Type prestoType, Object prestoNative)
+    {
+        if (prestoNative == null) {
+            return null;
+        }
+        else if (prestoType instanceof DecimalType) {
```

Comment:
nit: per the code style, we do not write `else` when not necessary, ie when the `then` block always returns.

I would remove `else` and also add blank space before subsequent `if` (here & for all cases below)